### PR TITLE
NO-JIRA: fix url in README file 

### PR DESCRIPTION
**Pull request by ptalgulk01**
This PR fixes broken link in the README file by updating them with the correct URL and updating few a note for not working one.

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by ptalgulk01** [comment]
@jhadvig can please add require label so it can get merged?

**Comment by jerolimov** [comment]
Hi @ptalgulk01, @jhadvig and @vikram-raj,

can we please fix the information (or remove it not needed anymore) instead of saying that they are deprecated, in the main ODC README. Thanks a lot. :)

/cc @jhadvig @vikram-raj 
/unassign @jhadvig 
/assign @vikram-raj 
/approve cancel

File: frontend/packages/dev-console/README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ You'll need to install the `OpenShift Pipeline Operator` found in the Admin sect
 
 There is a helpful script to install a couple Pipelines found in the `yamls` folder.
 
-Link to the [Script](./yamls/pipelines/install_pipeline_mocks.sh)
+Link to the [Script](./yamls/pipelines/install_pipeline_mocks.sh) 
+(The above link is now deprecated)
```
Comment (jerolimov) [inline frontend/packages/dev-console/README.md:12]: We should fix or remove information that are not valid anymore. IMHO, we don't want mark them as deprecated in our main README.

File: frontend/packages/helm-plugin/integration-tests/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Getting Started
 
 - Guidelines related to Setup, Standards, Review process are present in [README.md](frontend/packages/dev-console/integration-tests/README.md)
+(The above link is now deprecated)
```
Comment (jerolimov) [inline frontend/packages/helm-plugin/integration-tests/README.md:4]: Why are this information deprecated? Can't we link then the right one?

File: frontend/packages/helm-plugin/integration-tests/README.md [hunk 3]
```diff
@@ -54,7 +56,7 @@ frontend/packages/helm-plugin/integration-tests/
 
 Feature file - "regression" suite - execution from Cypress Dashboard
 
-1. Update the TAGS under env section in config file [Cypress.json file](frontend/packages/helm-plugin/integration-tests/cypress.json) as
+1. Update the TAGS under env section in config file [Cypress.json file](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/integration-tests/cypress.config.js) as
```
Comment (jerolimov) [inline frontend/packages/helm-plugin/integration-tests/README.md:59]: Shouldn't we prefer a relativ path?

**Review by jerolimov — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/README.md (hunk 1 above)
Comment (ptalgulk01) [reply frontend/packages/dev-console/README.md:12]: even here the provided path does not exist

**Comment by ptalgulk01** [comment]
@jerolimov can you please provide feedback about this?

**Review by vikram-raj — COMMENTED** [review]
Adding LGTM for this change. But we need to update and clean the dev-console readme.

/lgtm

**Pull request merged**